### PR TITLE
Pending changes exported from your codespace

### PR DESCRIPTION
**Pull request by Shmok9**

### CODE REVIEW

**Review by Perry2186 — APPROVED** [review]

**Review by Shmok9 — COMMENTED** [review]
#47

**Comment by Shmok9** [comment]
2

**Comment by Shmok9** [comment]
Shmok9:codespace-humble-halibut-g44gr4jpxw5jhpxq

**Comment by Shmok9** [comment]
1

**Comment by Shmok9** [comment]
1

**Review by Greeneislife — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+    "": {
+      "license": "MIT",
+      "dependencies": {
+        "season": "~>5.0"
+      }
+    },
+    "node_modules/async": {
+      "version": "1.5.2",
+      "resolved": "https://registry.npmjs.org/async/-/async-1.5.2.tgz",
+      "integrity": "sha512-nSVgobk4rv61R9PUSDtYt7mPVB2olxNR5RWJcAsH676/ef11bUZwvu7+RGYrYauVdDPcO519v68wRhXQtxsV9w=="
```
Comment (AungkoMCM) [inline package-lock.json:15]: https://g.dev/aungko-mcm

**Review by AungkoMCM — APPROVED** [review]
#

**Pull request closed without merging**